test/hotspot/gtest/runtime/test_cgroupSubsystem_linux.cpp

```diff
@@ -34,9 +34,8 @@
 
 #include <stdio.h>
 
-#ifdef MUSL_LIBC
-#include <libgen.h> // for basename
-#endif
+// for basename, needed for Alpine
```

Comment:
Nit: Drop `needed for Alpine`. We include `libgen.h` unconditionally so that the posix version of `basename` is being used (rather than the GNU version).